app/pdfs/distribution_pdf.rb

```diff
@@ -212,4 +215,22 @@ def non_request_data
         @distribution.line_items.total,
         ""]]
   end
+
+  def hide_columns(data)
+    column_names_to_hide.each do |col_name|
+      col_index = data.first.find_index(col_name)
+      data.each { |line| line.delete_at(col_index) } if col_index.present?
+    end
+    data
```

Comment:
There isn't really a reason to return `data` here since the operation is mutating the object itself. You can just call it like a method.

Reply:
Thanks, just updated that.